src/mixins/_forms.scss

```diff
@@ -139,19 +139,35 @@
   $fd-radio-label-padding: 0.3125rem;
   $fd-radio-focus-offset: 0.125rem;
 
-  @include fd-focus() {
-    &::before {
-      content: "";
-      position: absolute;
-      border: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
-      top: $margin - $fd-radio-focus-offset;
-      left: $margin - $fd-radio-focus-offset;
-      bottom: $margin - $fd-radio-focus-offset;
-      right: $fd-radio-label-padding - $fd-radio-focus-offset;
-      @content;
-    }
+  &::after {
+    content: "";
+    position: absolute;
+    border: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
+    top: $margin - $fd-radio-focus-offset;
+    left: $margin - $fd-radio-focus-offset;
+    bottom: $margin - $fd-radio-focus-offset;
+    right: $fd-radio-label-padding - $fd-radio-focus-offset;
+    @content;
+  }
+}
 
-    box-shadow: none;
-    outline: none;
+@mixin fd-form-label() {
+  @include fd-reset();
+  @include fd-ellipsis();
+
+  width: fit-content;
```

Comment:
`width: fit-content;` is not supported in IE11 and Edge.
If label is always going to be flex child I recommend to use `align-self: flex-start`